scripts/stats/cam/exevs_refs_grid2obs_stats.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/ksh
+#####################################################################
+# Purpose:   Setup some paths and run refs grid2obs stat ush scripts
+# 
+# Last updated 
+#              08/23/2024: by  Binbin Zhou, Lynker@EMC/NCEP
+#####################################################################
+set -x
+
+export machine=${machine:-"WCOSS2"}
+export WORK=$DATA
+cd $WORK
+
+#lvl = profile or sfc or both
+export lvl='both'
+
+#  verify_all = yes:  verify both profile and sfc (system + product)
+#  if lvl is not both, verify_all = no
+export verify_all=${verify_all:-'yes'}
+
+export prepare='yes'
+export verif_system='yes'
+export verif_profile='yes'
+export verif_product='yes'
+export gather=${gather:-'yes'}
+export verify=$VERIF_CASE
+export run_mpi=${run_mpi:-'yes'}
+
+#*************************************
+#check input data are available:
+#*************************************
+source $USHevs/$COMPONENT/evs_check_refs_files.sh
+export err=$?; err_chk
+if [ -e $DATA/verif_all.no ] ; then
+ export prepare='no'
+ export verif_system='no'
+ export verif_profile='no'
+ export verif_product='no'
+ export gather='no'
+ echo "Either prepbufr or REFS forecast files do not exist, skip grid2obs verification!"
+fi 
+
+export COMREFS=$COMINrefs
+export PREPBUFR=$COMINobsproc
+
+export GATHER_CONF_PRECIP=$PRECIP_CONF
+export GRID2OBS_CONF=$PARMevs/metplus_config/$STEP/$COMPONENT/grid2obs
+export GATHER_CONF_GRID2OBS=$GRID2OBS_CONF
+export MET_CONFIG=${METPLUS_BASE}/parm/met_config
+export maskpath=$MASKS
+export vday=$VDATE
+
+#  domain = conus or alaska or all
+export domain="all"
+
+export COMOUTrestart=$COMOUTsmall/restart
+[[ ! -d $COMOUTrestart ]] &&  mkdir -p $COMOUTrestart
+[[ ! -d $COMOUTrestart/prepare ]] &&  mkdir -p $COMOUTrestart/prepare
+[[ ! -d $COMOUTrestart/prepare/prepbufr.${vday} ]] &&  mkdir -p $COMOUTrestart/prepare/prepbufr.${vday}
+[[ ! -d $COMOUTrestart/system ]]  &&  mkdir -p $COMOUTrestart/system
+[[ ! -d $COMOUTrestart/profile ]] &&  mkdir -p $COMOUTrestart/profile
+[[ ! -d $COMOUTrestart/product ]] &&  mkdir -p $COMOUTrestart/product
+
+#************************************************
+# Prepare REFS member files just for sfc fields
+#************************************************
+if [ $prepare = yes ] ; then
+  $USHevs/cam/evs_refs_prepare.sh sfc
+fi
+
+#***************************************
+# Prepare the prepbufr data
+# **************************************
+if [ $prepare = yes ] ; then
+
+  if [ -s $COMINobsproc/rap.${VDATE}/rap.t12z.prepbufr.tm00 ] && [ -s $COMINobsproc/gdas.${vday}/00/atmos/gdas.t00z.prepbufr ] ; then
+
+     $USHevs/cam/evs_refs_prepare.sh prepbufr $domain
+     export err=$?; err_chk
+     $USHevs/cam/evs_refs_prepare.sh gdas_prepbufr
+     export err=$?; err_chk
+
+  else
+       export subject="GFS or RAP Prepbufr Data Missing for EVS ${COMPONENT}"
+       export MAILTO=${MAILTO:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
Please remove Geoff's email

Reply:
Ah sorry could we add it so `andrew.benjamin@noaa.gov` is included in this list?